CHANGES.rst

```diff
@@ -13,6 +13,8 @@ Version 0.9.0 (2022-12-13)
 - Fixes wrong numbering of restarted workflows by limiting the number of times a workflow can be restarted to nine.
 - Fixes ``Workflow.get_workspace_disk_usage`` to always calculate disk usage rather than relying on the quota usage values from the database, since these may not be up-to-date depending on the global quota update policy.
 - Fixes helper function that retrieves workflows by UUID to also additionally check that the provided user is the owner of the workflow.
+- Fixes the conversion to a human-readable format of negative resource amounts.
```

Comment:
- Fixes disk quota updater to prevent setting negative disk quota usage values.
- Fixes conversion of possibly-negative resource usage values to human-readable formats.